src/components/cap-reporter.js

```diff
@@ -66,6 +66,7 @@ export default class CapReporter extends LitElement {
 				font-weight: 600;
 				font-size: var(--font-size-250);
 				position: relative;
+				margin-bottom: 0.5rem;
```

Comment:
Here's another place we can rewrite the margin property to better support internationalization, and make use of the css variable